### ISSUE TITLE
Problème de lien vers les projets

### ISSUE THREAD

**Issue by Offirmo** [themesdefrance/Etendard#415]
Sur mon site de test : http://demoetendard-ef001.rhcloud.com/

Cliquer sur un projet échoue : le lien sur la page est `/project/projet-1-fr/` (provoque une 404) au lieu de `/projet/projet-1-fr/`

**Comment by themesdefrance** [comment]
Bonjour,
Avez-vous actualisé les permaliens dans Réglages > Permaliens ?

**Comment by Offirmo** [comment]
Effectivement, les permaliens sont réglés sur "Nom de l’article	http://demoetendard-ef001.rhcloud.com/exemple-article/"

**Comment by Offirmo** [comment]
Pas bon ?

**Comment by themesdefrance** [comment]
Je viens de comprendre, il faut changer le slug du CPT en fonction de la langue.

Il faut créer une fonction qui, selon la langue, renvoie le bon slug au niveau du filtre : etendard_portfolio_slug

**Comment by Offirmo** [comment]
Ce qui est étrange, c'est que ça ne fonctionne ni en français, ni en anglais...

**Comment by themesdefrance** [comment]
Dans ce cas, je veux bien un accès à votre site pour jeter un oeil.

**Comment by Offirmo** [comment]
accès envoyé par mail

**Comment by themesdefrance** [comment]
Je viens de tester et les projets fonctionnent en français..
Pour changer l'url en fonction de la langue, il faut utiliser une fonction pour tester la langue. Malheureusement, cela n'est pas compris dans le support d'Étendard.

**Comment by Offirmo** [comment]
Les projets fonctionnent en français ? De quelle manière ? 

Si je vais sur la home en français http://demoetendard-ef001.rhcloud.com/ et que je clique sur un projet -> 404

**Comment by themesdefrance** [comment]
Je viens de faire un modification dans functions.php et ça fonctionne. Par contre, les projets en anglais auront projet au lieu de project dans leur adresse
